modules/core/react/lib/theming/theme.ts

```diff
@@ -0,0 +1,63 @@
+import colors from '@workday/canvas-colors-web';
+import deepmerge from 'deepmerge';
+import {CanvasTheme} from './types';
+
+/**
+ * Considerations:
```

Comment:
Lots of question around what we want to include in the theme object and how we want to calculate some of the colors. Needs more discussion.

Reply:
We should also consider the use of more transparency for flexibility in the future (dark modes, overlapping elements like selection control ripples, etc.)